src/main/proto/inter_node_rpc_service.proto

```diff
@@ -95,6 +95,9 @@ enum MetricEnum {
   CACHE_EVICTION = 10 [(additional_fields).name = "cache eviction", (additional_fields).description = "cache eviction count"];
   CACHE_HIT = 11 [(additional_fields).name = "cache hit", (additional_fields).description = "cache hit count"];
   CACHE_MAX_SIZE = 12 [(additional_fields).name = "cache max size", (additional_fields).description = "max cache size in bytes"];
+
```

Comment:
Let's move this to under `JVM` resources, will mean other will get shifted but that if okay.